src/display.js

```diff
@@ -0,0 +1,38 @@
+const fetchFilm = (id) => fetch(`https://api.tvmaze.com/lookup/shows?tvrage= ${id}`);
+
+const movieCont = document.querySelector('.movie-cont');
+
+const displayMovies = (movieName, image) => {
+  movieCont.innerHTML += `<li>
+    <img src=${image} alt='' class='image'>
+    <div class='desc'>
+        <h2 class= 'item_id'>${movieName}</h2>
+        <div class='likes'>
+            <i class='far fa-heart'></i>
+            <p><span class='item-likes'>5</span></span><span>likes</span>
+            </p>
+        </div>
+    </div>
+    <div class='btn-cont'>
+        <button class='comment-btn' data-id='comment'>comment</button>
+    </div>
```

Comment:
- [ **OPTIONAL** ] Consider changing the `div` tags into `section` / `article`. This is for better SEO listings and makes your code more [semantic and meaningful](https://www.w3schools.com/html/html5_semantic_elements.asp) especially to other programmers.